### ISSUE TITLE
Group Chat

### ISSUE THREAD

**Issue by 0TYoung0** [0TYoung0/YearProject#1]
Use this issue for basic talk.

**Comment by daneelrakow** [comment]
Since you are the owner, can you go to settings and create a github pages site from the `pages` branch I made?

Thanks

**Comment by daneelrakow** [comment]
Are you gonna be good for Monday?

**Comment by 0TYoung0** [comment]
Yeh, ill be fine, just had some family issues to take care of.

**Comment by daneelrakow** [comment]
Okay good, hope everything is okay. Umm, don't think you missed anything big. We did presentations in comp sci and started the christmas cards and in english we did poetry

**Comment by 0TYoung0** [comment]
Alright cool, and things are maybe okay now ? lol.

**Comment by daneelrakow** [comment]
So I wanna get the logistics for tomorrow out of the way. Did you check to make sure you can go?

**Comment by 0TYoung0** [comment]
Yes I did, I can go

**Comment by daneelrakow** [comment]
Okay sweet, so I'm thinking 5pm-ish I come and get you?

**Comment by 0TYoung0** [comment]
Yeah. That's fine. 3320 144th St. West, Rosemount, Minnesota Sorry for the late reply. I went to Billy's for a late lunch in St. Paul then did some family shopping.

**Comment by 0TYoung0** [comment]
rsmtirish@gmail.com
Is your email right? I'm going to send you a lengthy paragraph maybe lol.

**Comment by daneelrakow** [comment]
Yeah, it's right. Are you busy this afternoon?

**Comment by daneelrakow** [comment]
Just respond via email

**Comment by 0TYoung0** [comment]
Uhm, yeh I'm going to Olive Garden at five I believe. Sadly....

**Comment by daneelrakow** [comment]
I mean it's 3 now...so

**Comment by 0TYoung0** [comment]
Lol yeh... I had been in the living room for a while. What did you need/want?

**Comment by daneelrakow** [comment]
I've been playing battlefront all day and wanna get out of the house lmao

**Comment by 0TYoung0** [comment]
lmao got it, well maybe sometime this week if you can think of anything to do. I need to get out of the house, which is also why I was going to send you an email. It got deleted from my draft box though, so I may just re write it here.

**Comment by daneelrakow** [comment]
You said you were leaving around 5 so doesn't that leave you with some time though?

**Comment by 0TYoung0** [comment]
Yes two hours, I don't know what I would do though? I'm going to re type what I attempt at sending you yesterday

**Comment by daneelrakow** [comment]
we could get food and you tell me in person?

**Comment by 0TYoung0** [comment]
So, I broke up with my girlfriend, and she still wants to be best friends, and talk to each other, but she can't talk to me without being extremely vague, and still wont really talk to me. She replies like once every two hours or so....

-- I ate 8 pieces of French toast (basically a full loaf of bread) and 12 pieces of bacon an our ago for a late breakfast... --- 

Now today she took a rain check on going ice skating, which to the point I'm just going to return mine then.... 100$ is not going to just sit in my room and never be used.... I have stuff I could buy that would actually be an investment.

**Comment by 0TYoung0** [comment]
She's taken 3 rain checks already, and wont talk to me face to face, and it's kinda pissing me off

**Comment by 0TYoung0** [comment]
BUT SHE STILL WANTS TO BE BEST FRIENDS, AND TELL EACH OTHER EVERYTHING

**Comment by daneelrakow** [comment]
when did you break up with her

**Comment by 0TYoung0** [comment]
ok so that was topic number one, topic number two, I was going to ask if you trusted me, as I have huge trust issues (even worse now because of this

**Comment by 0TYoung0** [comment]
Tuesday

**Comment by daneelrakow** [comment]
Okay and yeah I trust you I suppose. I don't really fully trust anyone and nor should anyone because that's retarded, but I trust you as much as I can, no offense.

**Comment by daneelrakow** [comment]
dont take that the wrong way

**Comment by 0TYoung0** [comment]
We broke up because I wasn't "the one", and she couldn't give me a diffinetive answer besides that. 
and no vise versa lol, I don't trust anyone either, I'd just like someone that would at least have my back, yehno? I'm pretty much done with all the kids at school thinking everything is perfectly fine when it's not...

**Comment by daneelrakow** [comment]
Tyler this doesn't just go for you, but for everyone in my life. I will always be there for almost anyone regardless of how they treat me. I think we've gone over this. Honestly, if Sydney or a few other people who have messed me up came to me for help, I would help them without hesitation. It bugs me, but yeah.

**Comment by 0TYoung0** [comment]
kk, was just checking. I needed someone to talk to who isn't 20+ years older than I was. lol.

**Comment by daneelrakow** [comment]
wanna like go for a walk or something...or code... idk

**Comment by daneelrakow** [comment]
walking and talking usually works

**Comment by 0TYoung0** [comment]
Also, third thing I was sending in the email. We have the presentation on the 17th I believe, I was wondering what you wanted to do, the thing in css? or something. I need something to focus my mind on. Everyone keeps asking me what I want to do and I always tell them I don't know, because I actually don't know. The only two things I'm interested in is graphic design and coding, but I haven't really been focusing on it

**Comment by daneelrakow** [comment]
^^

**Comment by 0TYoung0** [comment]
I think coding right now would be a better option... We used to walk and talk... and I run 2-4 miles a day, ~19 miles a week, so running/walking don't do much to relieve stress LOL

**Comment by daneelrakow** [comment]
I got a new laptop so...

**Comment by 0TYoung0** [comment]
nice, which one?

**Comment by daneelrakow** [comment]
it's a Lenovo Ideapad 310. 2.3gHz i3, 6gb ram and 1tb hdd. It gets the job done. Plus I got $230 off.

**Comment by 0TYoung0** [comment]
Dang nice, and yeh. 1TB + 6gb ram is perfect I think for coding, and it still has 2.3ghz which is about the same speed as the macs.

**Comment by daneelrakow** [comment]
I'm gonna put in some more ram soon though, I'll have 12gb lol

**Comment by 0TYoung0** [comment]
Also, I don't know if you've heard of a messaging app called Line
You can download it on PC with windows 8/10, I just installed it now, as I'm making a new account just to update a few things. 
It's pretty formal, and nice to have, and won't spam here or our emails

**Comment by 0TYoung0** [comment]
I have it on my phone too.

**Comment by 0TYoung0** [comment]
It's basically like a professional version (I mean depending on your group chat or the people your friends with) of kik

**Comment by daneelrakow** [comment]
downloading it now lol

**Comment by 0TYoung0** [comment]
cool 🗡

**Comment by daneelrakow** [comment]
so do you wanna do anything or what. we could go drive or something

**Comment by 0TYoung0** [comment]
Uhmmmm let me check to see If I actually have to go to Olive Garden, it's for a birthday party (ish, it's my sisters and her best friends as they were born on the same day) I'd rather not have to be time crunched unless your okay with that?

**Comment by daneelrakow** [comment]
BRUH OMG my dad didn't close his porn tab...................................uhh yeah sure

**Comment by 0TYoung0** [comment]
LOL, I don't think dad's realize that.... My uncle had four of them saved to his favorites... and he wondered why he had so many viruses

**Comment by daneelrakow** [comment]
if anything we'd have like an hour tho

**Comment by daneelrakow** [comment]
i got line btw

**Comment by 0TYoung0** [comment]
kk whats your username

**Comment by daneelrakow** [comment]
rsmtirish

**Comment by 0TYoung0** [comment]
ugh go to settings, and then go to ID, and set your ID as that

**Comment by 0TYoung0** [comment]
as well

**Comment by daneelrakow** [comment]
done
